### DISCUSSION TITLE
When I save state, it saves with 01/01/1970 date

### DISCUSSION THREAD

**Discussion by santo998** [visualboyadvance-m/visualboyadvance-m#1264] [General]
I think my emulator's date overflowed some way.

When I save state, it saves it as 01/01/1970.

I tried different games, and the same happens. So I think the problem is on VisualBoyAdvance.

Any clues on how to fix this?

**Comment by ZachBacon** [comment]
Keeping this open.

**Reply by santo998** [reply]
@ZachBacon can you do some **real work** and fix this?

Or are you here just to exercise "power" over users?

**Reply by ZachBacon** [reply]
How about you stop complaining, all the work we do here is volunteer.

**Reply by Squall-Leonhart** [reply]
There isn't a bug, this can only happen due to a situation involving your storage environment, the date and time of files is written by the FS api which we don't manipulate at all, we ask it to write a file, it writes the file.

**Comment by santo998** [comment]
@Squall-Leonhart ty so much!

You are right, it was due saving it on my network disk, which uses an outdated SAMBA version.

@ZachBacon you can learn from @Squall-Leonhart 🤣🤜🏻😭

**Reply by Squall-Leonhart** [reply]
I'd rather he didn't, i'm the more volatile one.

**Reply by ZachBacon** [reply]
> @Squall-Leonhart ty so much!
> 
> You are right, it was due saving it on my network disk, which uses an outdated SAMBA version.
> 
> @ZachBacon you can learn from @Squall-Leonhart 🤣🤜🏻😭

I'm the more sane, keep things on track guy, Squall like he mentioned is the volatile guy.
